spec/openapi.yaml

```diff
@@ -10151,6 +10151,16 @@ components:
           description: User identifier who created issue
           example: 661801f19693
           type: string
+        issueNumbers:
```

Comment:
Same comment as above - issueNumbers sounds like a number of issues associated with some object. If it is the unique ID of an issue or something like that, it should be called issueID or something along those lines?